android/app/src/main/kotlin/coop/polypoly/polypod/polyOut/PolyOut.kt

```diff
@@ -10,11 +10,10 @@ import java.nio.ByteBuffer
 open class PolyOut(
     val context: Context
 ) {
-    private val fsPrefix = "polypod://"
     private var readdirCache = mutableMapOf<String, Array<String>>()
     private var statCache = mutableMapOf<String, MutableMap<String, String>>()
     companion object {
-        private val fsPrefix = "polypod://"
+        val fsPrefix = "https://appassets.androidplatform.net/"
```

Comment:
Now I got lost. Sorry. A fs preffix starting with https? 
Mind you, I'm totally ignorant of what's going on here. I'm just surprised it's going from the polypod:// namespace to https. Also, the literal is used in a couple of places more, one an HTML file (probably not much can be done about it), but the other is FeatureContainer.kt. A module-wide constant would probably be better (and easier to refactor)

Reply:
This basically returns _direct_ HTTP URLs instead of our (made up) `polypod:` URLs.

This is probably my biggest comment to the PR, but _I would not expose HTTP URLs directly to the feature_, that's quite the leaky abstraction. I would handle it entirely in `RemoteClientPod`, more specifically within its `readFile` implementation.